src/puppetlabs/puppetdb/utils.clj

```diff
@@ -1,14 +1,19 @@
 (ns puppetlabs.puppetdb.utils
   (:require [puppetlabs.kitchensink.core :as kitchensink]
+            [puppetlabs.puppetdb.scf.storage-utils]
             [clojure.tools.logging :as log]
+            [clojure.string :as string]
             [schema.core :as s]
             [clojure.data :as data]
             [puppetlabs.puppetdb.schema :as pls]
+            [clojure.set :as set]
             [puppetlabs.puppetdb.archive :as archive]
             [clojure.java.io :as io]
+            [puppetlabs.puppetdb.scf.storage-utils :as sutils]
             [clojure.walk :as walk]
             [slingshot.slingshot :refer [try+]])
-  (:import [java.net MalformedURLException URISyntaxException URL]))
+  (:import [java.net MalformedURLException URISyntaxException URL]
+  [org.postgresql.util PGobject]))
```

Comment:
Alignment here, but also we might want to just move this to a proper schema in `pls` since we're using it in a lot of namespaces as such.